librz/asm/arch/java/jvm.c

```diff
@@ -7,7 +7,7 @@
 
 #define fail_if_no_enough_buffer_or_set(bytecode, jvm, n) \
 	if ((jvm->size - jvm->current) < n) { \
-		eprintf("%u < %u", jvm->size - jvm->current, n); \
+		RZ_LOG_DEBUG("java: not enough buffer (available: %u, needed: %u)\n", jvm->size - jvm->current, n); \
```

Comment:
`buffer is not big enough` or `buffer is out of memory` would be a better-looking message.